### PR TITLE
fix: post optionality and removal of tweet jobs

### PR DESCRIPTION
**Pull request by rebelchris**
Change post optionality:
- title is now optional
- sourceId is now required -> default `unknown` (added into the migration)
- tweeted is not gone

Also cleaned up all the tweeted related crons and workers, including package used for tweeting.

WT-1126 #done

### CODE REVIEW

File: src/migration/1678256072490-PostOptionality.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    );
+    await queryRunner.query(
+      `INSERT INTO "public"."source" ("id", "name", "handle", "private") VALUES ('unknown', 'unknown', 'unknown', 'true') ON CONFLICT DO NOTHING`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "post" ALTER COLUMN "sourceId" SET DEFAULT 'unknown'`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "post" ALTER COLUMN "sourceId" SET NOT NULL`,
+    );
```
Comment (sshanzel) [inline src/migration/1678256072490-PostOptionality.ts:19]: I think we already have a migration for this: https://github.com/dailydotdev/daily-api/blob/main/1677643070022-UnknownSource.ts

Comment (rebelchris) [reply src/migration/1678256072490-PostOptionality.ts:19]: Ah I thought so, but it won't run indeed from main so I didn't see it reflect in my DB

Comment (sshanzel) [reply src/migration/1678256072490-PostOptionality.ts:19]: Right. Feel free to delete the one in the root folder then I will close the PR itself.

**Review by sshanzel — CHANGES_REQUESTED** [review]
Thanks for removing all the twitter-related unused codes. Kindly delete the migration file then this should be good to go 🚢

File: src/migration/1678256072490-PostOptionality.ts (hunk 1 above)
Comment (rebelchris) [reply src/migration/1678256072490-PostOptionality.ts]: Done 🙏

**Review by idoshamun — CHANGES_REQUESTED** [review]
Please make sure to delete the workers from the worker array, the same applies to the crons. Regarding the cron, I didn't see you removed it from pulumi

**Comment by rebelchris** [comment]
@idoshamun I think all of them where already removed from the worker array and pulumi when we experienced the Twitter issues.
Unless I missed something?

**Comment by idoshamun** [comment]
@rebelchris you're right! 🤦‍♂️

**Review by idoshamun — APPROVED** [review]

**Pull request merged**